packages/itwinui-css/src/select/select.scss

```diff
@@ -48,6 +48,33 @@
   &.iui-placeholder {
     color: var(--iui-color-text-disabled);
   }
+
+  @each $statusColor in informational, positive, warning, negative {
```

Comment:
possible to avoid duplication from input.scss?

Reply:
iui-input-status that has data attr inside?

Reply:
yes, or even just `data-iui-input-status` attribute

Reply:
Did this. Specificity required to have strange selectors.